plugin/src/main/java/at/helpch/chatchat/event/ChatChatEvent.java

```diff
@@ -0,0 +1,71 @@
+package at.helpch.chatchat.event;
```

Comment:
should probably move this class to the API module instead

Reply:
Ok I might be dumb. I swear at first I've put it there. dk why I decided to move it.

Actually I might know why lol. I used ChatFormat instead of format and didn't want to add the plugin module as a dependency on the api module... I forgot I should use Format instead.